### PR TITLE
restructure readout modules schema to follow closer daq module code o…

### PR DESCRIPTION
**Pull request by glehmannmiotto**
…rganization.

Work is not complete as the views files need to be updated and the diagrams recreated for the documentation.
I also need advice from Roland on which of the parameters belong to any request handler and which ones are specific to only some request handler configurations (and same for latency buffers).

### CODE REVIEW

File: schema/appdal/application.schema.xml [hunk 1]
```diff
@@ -222,16 +233,33 @@
...
+
+ <class name="BinarySearchLatencyBuffer">
+  <superclass name="LatencyBuffer">
+ </class>
+
+ <class name="SkipListLatencyBuffer">
+  <superclass name="LatencyBuffer">
+ </class>
+
+ <class name="ReadoutModelConf">
```
Comment (gcrone) [inline schema/appdal/application.schema.xml:236]: ReadoutModelConf or ReadoutModuleConf?

**Review by gcrone — APPROVED** [review]
Looks good. I knew there were some things that didn't quite match between the drawio diagram from the summer and the actual module code!

**Comment by gcrone** [comment]
And of course I forgot to fix the network connection descriptor after changing coredal!!

File: schema/appdal/application.schema.xml (hunk 1 above)
Comment (glehmannmiotto) [reply schema/appdal/application.schema.xml:236]: the class is Model, not Module in the c++ readout code.

**Pull request merged**